### DISCUSSION TITLE
3rd party IDP

### DISCUSSION THREAD

**Discussion by LauraGOP** [monarc-project/MonarcAppFO#473] [General]
Hello,

A client is currently implementing an instance of Monarc.
They would like to add a 3rd party IdP(Active Directory, LDAP or Azure AD).
I saw a conversation in January that Cases wanted to implement it soon.
Is it possible now ?

Thank you!

**Comment by ruslanbaidan** [comment]
Hello,

LDAP is not available in Monarc yet.
We have another priorities for the moment and after we can come back to the planned development.

**Reply by LauraGOP** [reply]
Hello,

LDAP is not, what about Azure AD and AD?

Thank you!

**Reply by ruslanbaidan** [reply]
MS Active Directory uses Lightweight Directory Access Protocol (LDAP).
Azure AD is not free and open source, so the support will not be added.

**Reply by LauraGOP** [reply]
Ok, thank you!

**Comment by yentl-creyf** [comment]
(A)AD is the microsoft implementation of Lightweight Directory Access Protocol (LDAP) .
Other protocols such as Oauth 2.0 / OpenID  or SAML should be possible
Related: 
#446 
#447
